docs/readthedocs/source/doc/UserGuide/databricks.md

```diff
@@ -87,10 +105,144 @@ Output on Databricks:
 
 ![](images/init-orca-context.png)
 
+**Run ncf_train example on Databricks**
+
+create a notebook and run the following example.
```

Comment:
create -> Create
Add `Note that to make things simple, we are just generating some dummy data for this example`